msal/tests/Test.MSAL.NET.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -1300,71 +1297,6 @@ public void B2CAcquireTokenTest()
                 Assert.IsNotNull(result.Account);
             }
         }
-
-        [TestMethod]
-        [TestCategory("PublicClientApplicationTests")]
-        public void B2CAcquireTokenWithValidateAuthorityTrueTest()
-        {
-            using (var httpManager = new MockHttpManager())
-            {
-                PublicClientApplication app = new PublicClientApplication(
-                    httpManager,
-                    new TelemetryManager(),
-                    CoreTestConstants.ClientId,
-                    CoreTestConstants.B2CAuthorityNotTrustedHost);
-                app.ValidateAuthority = true;
-
-                MockWebUI ui = new MockWebUI()
-                {
-                    MockResult = new AuthorizationResult(
-                       AuthorizationStatus.Success,
-                       CoreTestConstants.B2CAuthorityNotTrustedHost + "?code=some-code")
-                };
-
-                try
-                {
-                    AuthenticationResult result = app.AcquireTokenAsync(CoreTestConstants.Scope).Result;
-                }
-                catch (Exception exc)
-                {
-                    Assert.AreEqual(exc.InnerException.Message, CoreErrorMessages.UnsupportedAuthorityValidation);
-                }
```

Comment:
this should now work and not throw an exception, right? Why not keep the scenario?

Reply:
yep. i put them back. 

---
In reply to: [234388585](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1405#discussion_r234388585) [](ancestors = 234388585)